nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -34,17 +34,21 @@ Keep an old app running and swap the ports when the new one is booted.
           host = "*4";
           port = 6981;
         }];
+        rotate-logs= false;
       };
       # You want that ip-from-header in the nginx setup case
       # so it's not set to 127.0.0.1.
       # using a port above 1024 allows you to avoid needing CAP_NET_BIND_SERVICE
+      # rotate-logs = false; emits keter logs and it's applications to stderr
+      # which allows journald to capture them.
```

Comment:
L39 to 41 seemed to explain the defaultText options in a similar way, I followed this pattern.

Reply:
It's.. your own module? And therefore your own pattern. Please consult other modules.

Reply:
yeah, I've to admit, I don't remember what past me did or why, anyway, I moved it to L37.
there is no option description for this particular option since it just emits the global config as a yaml file and let's keter deal with parsing the options.